<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of the file</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_input_name_file">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_text">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Write whatever you want :)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_input_text">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>591</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ok">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>470</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_input_folder">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_directory">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse the folder where you want to save the file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bttn_browse">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
